src/app/modules/project/edit/edit.component.html

```diff
@@ -21,11 +21,11 @@
   <div class="offset-2 col-8">
     <h1>Edit project</h1>
 
-    <ng-container *ngIf="projectLoading; else projectLoaded">
+    <ng-container *ngIf="projectLoading">
```

Comment:
What's the reason for this change?

Reply:
I had to switch out all the templates from that page because else the FileComponent wouldn't bind in any of the lifecycle hooks. If anyone has a better approach to this I would be happy to change is, I don't like the way it is now either but it was the only way I got it to work.